src/components/header.js

```diff
@@ -1,37 +1,17 @@
-import React from 'react';
+import React from "react";
 
-import TopNav from './top-nav';
-import InfoModal from './info-modal';
+import TopNav from "./top-nav";
 
-import './header.css';
+import "./header.css";
 
-export default class Header extends React.Component  {
-    constructor(props) {
-        super(props);
-        this.state = {
-            showInfoModal: false
-        };
-    }
-
-    toggleInfoModal() {
-        this.setState({
-            showInfoModal: !this.state.showInfoModal
-        });
-    }
-
-    render() {
-        let infoModal;
-        if (this.state.showInfoModal) {
-            infoModal = <InfoModal onClose={() => this.toggleInfoModal()} />;
-        }
-
-        return (
-            <header>
-                <TopNav onInfo={() => this.toggleInfoModal()}
-                    onNewGame={this.props.onNewGame} />
-                {infoModal}
-                <h1>HOT or COLD</h1>
-            </header>
-        );
-    }
-};
+export default function Header(props) {
+  return (
+    <header>
+      <TopNav
+        onGenerateAuralUpdate={props.onGenerateAuralUpdate}
```

Comment:
Make these arrow functions so we can teach one rule without thinking about binding:
```js
onGenerateAuralUpdate={() => props.onGenerateAuralUpdate()}
```